src/test/scala/chiselTests/LayerSpec.scala

```diff
@@ -48,23 +48,29 @@ class LayerSpec extends ChiselFlatSpec with Utils with MatchesAndOmits {
     )()
   }
 
-  it should "allow for defines in a layer to drive layer-colored probes" in {
+  it should "allow for defines to layer-colored probes" in {
 
     class Foo extends RawModule {
-      val in = IO(Input(Bool()))
-      val a = IO(Output(Probe(Bool(), A)))
-      val b = IO(Output(Probe(Bool(), A.B)))
+      val a, b, c, d, e = IO(Input(Bool()))
+      val x = IO(Output(Probe(Bool(), A)))
+      val y = IO(Output(Probe(Bool(), A.B)))
+      define(x, ProbeValue(a))
+      define(y, ProbeValue(b))
       layer.block(A) {
-        define(a, ProbeValue(in))
+        define(x, ProbeValue(c))
+        define(y, ProbeValue(d))
         layer.block(A.B) {
-          define(b, ProbeValue(in))
+          define(y, ProbeValue(e))
         }
       }
     }
```

Comment:
This will produce illegal FIRRTL as the probes are defined more than once. However, it makes the test terser to write it this way.